### PR TITLE
adding fixes for failing production tests.

### PR DESCRIPTION
**Pull request by RubyGemTSL**
removed wip tag

removed unnecessary code.

### CODE REVIEW

File: _cucumber/features/support/hooks.rb [hunk 1]
```diff
@@ -59,6 +59,11 @@
 end
 
 def user_logout
-  visit("https://it-developers.stage.redhat.com/auth/realms/rhd/protocol/openid-connect/logout?redirect_uri=#{@redirect_url}%3Fredirect_fragment%3D!")
+  if Capybara.app_host.include?('stage') || Capybara.app_host.include?('docker')
```
Comment (ollyjshaw) [inline _cucumber/features/support/hooks.rb]: I think what your are really doing here is testing if the environment is live? If that's the case, then test for the live URL as it is clearer and will allow us to use other forms of non-live

Comment (RubyGemTSL) [inline _cucumber/features/support/hooks.rb]: No, this is a hack to logout. As we cant delete cookies using selenium.

Comment (RubyGemTSL) [inline _cucumber/features/support/hooks.rb]: This was also part of a previous PR, I just added the production URL in this PR.

Comment (ollyjshaw) [inline _cucumber/features/support/hooks.rb]: I'm suggesting that the code be changed to 

```
if Capybara.app_host.equals?('developers.redhat.com')
  visit("https://developers.redhat.com/auth/realms/rhd/protocol/openid-connect/logout....
else
  visit("https://it-developers.stage.redhat.com/auth/realms/rhd/protocol/openid-connect/logout....
```

logic is "for live, logout here. For everywhere else log out here"

**Comment by RubyGemTSL** [comment]
retest this please

**Comment by RubyGemTSL** [comment]
@paulrobinson just waiting for Blinkr then this can be merged? cc @ollyjshaw

**Comment by ollyjshaw** [comment]
:+1:

**Pull request merged**